### DISCUSSION TITLE
Add suggested file title to import and export settings file?

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#718] [Ideas]
It would be helpful to add a suggested file name when saving the backup .reg file with the cool new import/ export settings feature.

![Screenshot 2022-01-23 140524](https://user-images.githubusercontent.com/48499740/150682640-c5ab81ed-450f-4a0a-a71c-833097a318b0.png)

**Comment by valinet** [comment]
Thanks. Implemented that in 22000.466.41.15.
